### PR TITLE
feat(ironfish,cli): Change account import to only use spending key

### PR DESCRIPTION
**Pull request by mat-if**
## Summary

Since the public address size changed and we changed some of our key personalization strings, it was possibly to import an account that was in an invalid state by importing an incorrect (but technically valid) incoming and outgoing view key

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/wallet/account.ts [hunk 1]
```diff
@@ -19,6 +19,8 @@ import { WalletDB } from './walletdb/walletdb'
 
 export const ACCOUNT_KEY_LENGTH = 32
 
+export type AccountImport = { name: string; spendingKey: string }
```
Comment (mat-if) [inline ironfish/src/wallet/account.ts:22]: I'm not sure if we have a standardized pattern for re-usable types, so I wanted to specifically call this out - it feels kind of strange here but not sure if there's a better place for it.

**Review by jowparks — APPROVED** [review]
Tested in huddle, things are working as expected

File: ironfish/src/wallet/account.ts (hunk 1 above)
Comment (NullSoldier) [reply ironfish/src/wallet/account.ts:22]: well, I don't really think we need this type exactly but it's fine either way. When you use it in importFile we just cast the result if JSON parsing to this type, but in reality it should be unknown and fed through a YUP schema.

**Review by NullSoldier — APPROVED** [review]

**Comment by NullSoldier** [comment]
It works, I tested it by importing a file from phase 2

**Pull request merged**